networkx/generators/geometric.py

```diff
@@ -279,14 +432,19 @@ def geographical_threshold_graph(n, theta, alpha=2, dim=2, pos=None,
     nx.set_node_attributes(G, 'weight', weight)
     nx.set_node_attributes(G, 'pos', pos)
 
+    #if p_dist not supplied, use default r^alpha
+    if p_dist == None:
+        def p_dist(r):
+            return r**-alpha
+
     # Returns ``True`` if and only if the nodes whose attributes are
     # ``du`` and ``dv`` should be joined, according to the threshold
     # condition.
     def should_join(pair):
```

Comment:
I had to flip around the logic here to incorporate the custom p_dist function, so the default function is now r^-alpha instead of r^alpha. This seems more consistent with your original GTG paper @hagberg , but there's probably a reason you had the logic the way it was, so want to be sure to double check with you that this is ok.